src/modules/integrator-guide/pages/security-model/permissions-evaluation.adoc

```diff
@@ -0,0 +1,315 @@
+include::ROOT:partial$variables.adoc[]
+
+= Permission Evaluation
+
+The permission evaluation feature allows administrators to check which actions or operations can be performed by a user or a combination of roles.
+The evaluation entails comparing the different roles assigned to a user to determine the user's ability to perform the various operations within a resource.
+
+[NOTE]
+====
+xref:access-control/index.adoc[{PRODUCT_NAME} RBAC] uses the roles of the given user to determine if they have access to a given resource.
+====
+
+== How to evaluate applied permissions
+
+[NOTE]
+====
+Most of the {PRODUCT_NAME} resources allow you to define which xref:access-control/roles.adoc[roles] are allowed to access the resource.
+Locate the btn:[permissions] button in the user interface in order to manage the resource.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "security-model/access-control/general-permissions-button.png",
+    "alias": "general-permissions-button",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 320,
+    "y": 0,
+    "h": 550,
+    "w": 1600
+  },
+  "focus": {
+    "x": 1466,
+    "y": 80,
+    "h": 31,
+    "w": 206
+  },
+  "annotations": [{
+    "kind": "box",
+    "x": 1536,
+    "y": 80,
+    "w": 134,
+    "h": 30
+  }]
+}
+----
+====
+
+1. Navigate to the resource of choice, and click the [btn:[permissions].
```

Comment:
Don't do this ordered list-like pattern and instead do it like how you've done it for duplicate detection (no numbering, just a series of steps (also similar to how I've done mine here src/modules/integrator-guide/pages/compose-configuration/record-revisions.adoc).

Some reasoning:

1. when you do it like this, you can't follow [this convention](https://docs.cortezaproject.org/corteza-docs/2022.9/developer-guide/documentation/index.html#_ordered_lists) and it becomes harder to maintain (you also missed a step yourself -- `Click on the btn:[save] button.` has no number).
2. these numbers don't really do anything on a long list with screenshots -- prefer to not use them or make sub-sections